lib/actionkit_connector.rb

```diff
@@ -154,5 +170,51 @@ def parse_action_options(options)
       end
       included_options
     end
+
+    def validate_donation_options(options)
+      required_base_keys = [:donationpage, :order, :user]
```

Comment:
> You can use keyword arguments for this

I'm aware of that, but I was torn between doing so and the need to compile a hash of values after the fact. In this case, `donationpage` both needs to be present *and* needs to conform to a certain structure, and if you already have a valid hash to send to the method, it seems a bit mean to force someone to split that back out into subsections just so that I can compile it again for you later.

Thoughts?